python/pfnano_cff.py

```diff
@@ -68,46 +68,46 @@ def PFnano_customizeMC_noInputs(process):
 def PFnano_customizeData(process):
     addPFCands(process, False)
     add_BTV(process, False, keepInputs=['DeepCSV','DDX'])
-    process.NANOAODSIMoutput.fakeNameForCrab = cms.untracked.bool(True)  # needed for crab publication
+    process.NANOAODoutput.fakeNameForCrab = cms.untracked.bool(True)  # needed for crab publication
```

Comment:
In PR https://github.com/cms-jet/PFNano/pull/46 there was the proposal to not put these modifiers into the `cff` but instead give them with each `cmsDriver` command. I think both are fine, however we could unify the config generation and synchronize across branches as much as possible by moving to the `--customize_commands` variant here as well and denote that in the readme as well.

Reply:
(Same comment for all entries in `pfnano_cff.py`, not only for Data actually)